src/modules/select/wallets/meetone.ts

```diff
@@ -0,0 +1,38 @@
+import { extensionInstallMessage } from '../content'
+import { WalletModule, Helpers, CommonWalletOptions } from '../../../interfaces'
+
+import meetoneIcon from '../wallet-icons/icon-meetone.png'
+import meetoneIcon2x from '../wallet-icons/icon-meetone.png'
+
+function meetone(options: CommonWalletOptions): WalletModule {
+  const { preferred, label, iconSrc, svg } = options
+
+  return {
+    name: label || 'MEETONE',
+    iconSrc: iconSrc || meetoneIcon,
+    iconSrcSet: iconSrc || meetoneIcon2x,
+    svg,
+    wallet: async (helpers: Helpers) => {
+      const {
+        // createLegacyProviderInterface,
+        createModernProviderInterface
```

Comment:
Also de-structure the `getProviderName` property off of the helpers object as it will be needed in the edit below